test/src/test/java/SslTest.java

```diff
@@ -0,0 +1,60 @@
+import io.restassured.RestAssured;
+import org.hamcrest.core.IsNull;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.hamcrest.Matchers.*;
+
+
+public class SslTest extends AbstractFeatureServiceTest {
+
+    @Before
+    public void connectToSslPort() {
+        RestAssured.port = 8093;
```

Comment:
This should be modified to point to 443 and use "https" - which leads to a "PKIX path building failed" error - looks like we can use https://stackoverflow.com/a/46011747/3306099 to work around this in the test, which is fine. Actually - let's have a second test method that makes an https call without the above hack and verify that it fails - that would be a useful test to make sure SSL is setup correctly. 

I'm also thinking that server.js should be modified to say - either I expose the SSL port if it's defined and enabled, or I expose "port". Which means that if a user adds the SSL port and enables it, they don't need to specify "port" because it won't be used. 

Otherwise, we have a setup where both e.g. 443/https and 8093/http are available for usage, which doesn't seem like a sensible default (if a user wants this, they can always construct their own Koop server). 

Note that the config in default.json is fine since the SSL config isn't enabled. Our docs can tell the user - you can enable this, in which case you're free to remove "port" because the server won't care about it.